src/Http/Response.php

```diff
@@ -89,6 +91,14 @@ public function getBody(): StreamInterface
         return $this->response->getBody();
     }
 
+    /**
+     * If there is no stream, consider it an empty response.
+     */
+    public function isEmpty(): bool
```

Comment:
Should I see this being used elsewhere in this library, or is this for the client to use?

Reply:
Nothing is using it in the library. The client is using it as a way to check if a response is truly empty or not (this affects some of the integration work we will be discussing). This method provides a more universal way to say " The response is empty" than the various implementations the client has had to make before.